contracts/JBChainlinkV3PriceFeed.sol

```diff
@@ -41,7 +48,14 @@ contract JBChainlinkV3PriceFeed is IJBPriceFeed {
   */
   function currentPrice(uint256 _decimals) external view override returns (uint256) {
     // Get the latest round information. Only need the price is needed.
-    (, int256 _price, , , ) = feed.latestRoundData();
+    (uint80 roundId, int256 _price, , uint256 updatedAt, uint80 answeredInRound) = feed
+      .latestRoundData();
+    // Make sure the price isn't stale.
+    if (answeredInRound < roundId) revert STALE_PRICE();
```

Comment:
Shouldn't we rather check the last timestamp? ETH/USD is updated for 0.5% deviation or 3600 seconds for instance (maybe some have longer heartbeat tho)

![image](https://user-images.githubusercontent.com/83670532/178688864-8f69c53c-1e15-4543-a897-7af92fbb71e9.png)